CDP4JsonSerializer.Tests/JsonSerializerTestFixture.cs

```diff
@@ -617,19 +617,19 @@ private class TestPostOperation
             /// <summary>
             /// Gets or sets the collection of DTOs to delete.
             /// </summary>
-            [JsonProperty("_delete")]
+            [JsonPropertyName("_delete")]

```

Comment:
please use the `PostOperation` from the `CDP4DalCommon` library to test

Reply:
Implemented correctly now, inheriting abstract base class PostOperation.

Reply:
PostOperation does not know about Json Serialization, so it does not have JsonPropertyname attributes set. Therefore this specific TestPostOperation should stay.